scripts/al2022/install-workarounds.sh

```diff
@@ -3,7 +3,11 @@ set -ex
 
 # update to AL2022 release that has latest repos
 # (get the latest version from dnf check-release-update)
-sudo dnf update -y --releasever=2022.0.20220204
+sudo dnf check-release-update >info 2>&1
```

Comment:
could the `info` file be a tmp file? you can create one and get it's name with mktemp like this `infofile=$(mktemp)`.